apps/web/src/components/Messages/PreviewList.tsx

```diff
@@ -141,22 +142,25 @@ const PreviewList: FC<PreviewListProps> = ({ className, selectedConversationKey
               />
             </button>
           ) : (
-            sortedProfiles?.map(([key, profile]) => {
-              const message = messages.get(key);
-              if (!message) {
-                return null;
-              }
-
-              return (
-                <Preview
-                  isSelected={key === selectedConversationKey}
-                  key={key}
-                  profile={profile}
-                  conversationKey={key}
-                  message={message}
-                />
-              );
-            })
+            <Virtuoso
+              style={{ height: '100%' }}
```

Comment:
Do they support classNames? if so we can use tailwind class `h-full`

Reply:
It looks like it does. Just tested locally with `h-full` and it seems to work just fine.